REVIEW.md

```diff
@@ -0,0 +1,24 @@
+This document MUST be updated everytime a PR has been identified has the source of a bug in develop. The post mortem analysis MUST include: the CAUSE of the bug, the RESOLUTION PROCESS and PREVENTION MEASURES.
+
+
+# "sql: no rows in result set" error on iOS login
+
+## Cause
+
+There was errors in the iOS native code:
+- "loginWithKeycard" method was not implemented
+- "createAndLoginWithKeycard" method was actually used instead of "createAndLogin** which is used for regular account creation without keycard and doesn't have the same final argument
+
+QA process was focused on account creation and login with keycard, and this bug would only appear specifically on iOS when creating an account without keycard and only during the next login. It was therefore a more complex path to reach that what could have been expected from this PR, and it was caused in develop by a developer working on iOS simulator because that is a common path during development.
```

Comment:
```suggestion
QA process was focused on account creation and login with keycard, and this bug would only appear specifically on iOS when creating an account without keycard and only during the next login. It was therefore a more complex path to reach than what could have been expected from this PR, and it was caused in develop by a developer working on iOS simulator because that is a common path during development.
```